doc/release-process.md

```diff
@@ -12,10 +12,34 @@ It is best if the maintainers are present during the release, so they can help e
 
 This process also assumes that there will be no minor releases for old major releases.
 
+We aim to cut a regular release every 3-4 months, approximately twice as frequent as major Bitcoin Core releases. Every second release should be published one month before the feature freeze of the next major Bitcoin Core release, allowing sufficient time to update the library in Core.
+
+## Sanity Checks
+Perform these checks before creating a release:
+
+1. Ensure `make distcheck` doesn't fail.
+```shell
+make distcheck
+```
+2. Check installation with autotools:
+```shell
+dir=$(mktemp -d)
+./configure --prefix=$dir && make install && ls -l $dir/include $dir/lib
+gcc -o ecdsa examples/ecdsa.c -I $dir/include -L $dir/lib/ -l secp256k1 && ./ecdsa
+```
+3. Check installation with CMake:
+```shell
+dir=$(mktemp -d)
+mkdir build && cmake -B build -DCMAKE_INSTALL_PREFIX=$dir && cmake --build build --target install && ls -l $dir/include $dir/lib*
```

Comment:
```suggestion
mkdir build && cmake -B build -DCMAKE_INSTALL_PREFIX=$dir -DBUILD_SHARED_LIBS=OFF && cmake --build build --target install && ls -l $dir/include $dir/lib*
```

Reply:
I don't want to `rm -rf` the user's files.

Reply:
@real-or-random why wouldn't we want to build the shared library?

Reply:
Since https://github.com/bitcoin-core/secp256k1/pull/1230, our CMake builds are either shared or static (but not both), because that's more common and simpler in CMake. And variable `BUILD_SHARED_LIBS` is the CMake convention for this. So `-DBUILD_SHARED_LIBS=OFF` doesn't only enable static libs, it *enables* static builds. But 

But yeah, the `rpath` thing does the job as well and is probably a bit cleaner.

Reply:
> I don't want to `rm -rf` the user's files.

nit: If you want to touch this again, what about this variant for additional convenience?
```suggestion
build=$(mktemp -d)
cmake -B $build -DCMAKE_INSTALL_PREFIX=$dir && cmake --build $build --target install && ls -l $dir/include $dir/lib*```